examples/go.mod

```diff
@@ -0,0 +1,19 @@
+module go.mongodb.org/atlas-sdk/v20231115008/examples
+
+go 1.22.1
+
+require (
+	github.com/go-test/deep v1.1.0
+	github.com/mongodb-forks/digest v1.1.0
+    go.mongodb.org/atlas-sdk/v20231115008 v20231115008.1.0 
+	github.com/hashicorp/go-retryablehttp v0.7.5 
```

Comment:
```suggestion
	github.com/go-test/deep v1.1.0
	github.com/mongodb-forks/digest v1.1.0
	go.mongodb.org/atlas-sdk/v20231115008 v20231115008.1.0 
	github.com/hashicorp/go-retryablehttp v0.7.5 
```

Reply:
go mod tidy do not cover changes for some reasons.